### PR TITLE
add new message to update flag

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

https://github.com/thoth-station/thoth-application/issues/507

## This introduces a breaking change

- [ ] Yes
- [x] No

## This should yield a new module release

- [x] Yes
- [ ] No

## This Pull Request implements

This message directly corresponds with an update of a row of the python package version table which will be handled by investigator.

## Description

Add new message, included `value` so that the operation is reversible if necessary

### CODE REVIEW

File: thoth/messaging/update_is_downloadable_ppv.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+"""This is Thoth Messaging module for UpdateIsDownloadableMessage."""
+
+from .message_base import MessageBase, BaseMessageContents
+
+
+class UpdateIsDownloadableMessage(MessageBase):
```
Comment (pacospace) [inline thoth/messaging/update_is_downloadable_ppv.py:24]: what about `NotDownloadedPackageMessage` as it described what happened?

File: thoth/messaging/update_is_downloadable_ppv.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    topic_name = "thoth.security-indicator.update-is-downloadable"
+    _message_version = 1  # update on schema change
+
+    class MessageContents(BaseMessageContents, serializer="json"):  # type: ignore
+        """Class used to represent a contents of a update-is-downloadable message."""
+
+        package_name: str
+        package_version: str
+        index_url: str
+        value: bool
```
Comment (pacospace) [inline thoth/messaging/update_is_downloadable_ppv.py:36]: what goes into value? if this message is sent we know we have to set the flag to False in storages

Comment (KPostOffice) [reply thoth/messaging/update_is_downloadable_ppv.py:36]: I would like the ability for the message to be reversible.  It is possible to upload a source distro for a package after the fact and we may want to watch for that eventually

**Review by pacospace — APPROVED** [review]
Thanks!!

**Pull request merged**